### ISSUE TITLE
Show folder size

### ISSUE THREAD

**Issue by Shaerif** [doublecmd/doublecmd#99]
I would suggest adding this option as I've tried both beta and alpha, and it's not available

**Comment by JonnyTech** [comment]
Menu >> Commands >> Show Occupied Space (Alt+Shift+Enter)

**Comment by Skif-off** [comment]
Also see [DirSizeCalc](https://totalcmd.net/plugring/dirsizecalc.html) (Win) or [calcsize](https://github.com/j2969719/doublecmd-plugins/tree/master/plugins/wdx/calcsize) (Linux).

**Comment by antodc** [comment]
Hope someone can help me. I am trying to use DirSizeCalc, as suggested above by @Skif-off. I would like to use it as described [here](https://www.ghisler.ch/board/viewtopic.php?t=18021&hilit=unit&sid=96d5b97b44f0f6395842ba323c78641c&start=414). However, in DC the size is always shown in bytes, and the unit is never shown. If I check the "show unit" field, it gets unchecked automatically every time.

**Comment by Skif-off** [comment]
It looks like Double Commander can't work with the custum units in this case... What about the existing units?

As a last resort, try [filecountwdx.lua](https://github.com/j2969719/doublecmd-plugins/blob/master/plugins/wdx/scripts/filecountwdx.lua).

**Comment by antodc** [comment]
Wow! [filecountwdx.lua](https://github.com/j2969719/doublecmd-plugins/blob/master/plugins/wdx/scripts/filecountwdx.lua) works great. Thank you!!
Two more things:
- Is it possible to add the thousand separator for files/dirs count?
- Is it possible to show the size also for files, in addition to folders?

**Comment by antodc** [comment]
Found out an issue when using [filecountwdx.lua](https://github.com/j2969719/doublecmd-plugins/blob/master/plugins/wdx/scripts/filecountwdx.lua) and sorting by size. It seems that the unit is not taken into account, so that for instance 100 Kb is considered greater than 10 Mb.

**Comment by Skif-off** [comment]
> Two more things

No, I'm not sure that this can be implemented in the current form.

> It seems that the unit is not taken into account, so that for instance 100 Kb is considered greater than 10 Mb.

"Folder size (auto)"? The "Folder size (auto)" field returns a string, and in this case the data is sorted as strings.

**Comment by ags1234** [comment]
**An option to enable automatic display of folder sizes in Size column when in Full panel view mode!**

I am using this is FreeCommander and Speed Commander and it is very useful when working with a lot of directories.
Why won't it be implemented in DC?

<img width="487" height="242" alt="Image" src="https://github.com/user-attachments/assets/097b276a-a813-419d-818e-41477b2c155f" />
